workload/src/main/java/com/google/finapp/WorkloadMain.java

```diff
@@ -33,75 +36,66 @@
 import org.apache.commons.cli.ParseException;
 
 /**
- * A workload generator for the finance sample app that creates traffic by sending gRPC requests to
- * the server.
+ * An executable method for a workload generator for the finance sample app that creates traffic by
+ * sending gRPC requests to the server.
  */
 public final class WorkloadMain {
-  private static final String DEFAULT_ACCOUNT_BALANCE = "10000";
-  private static final String DEFAULT_TRANSFER_AMOUNT = "20";
   private static final Logger logger = Logger.getLogger(WorkloadMain.class.getName());
 
   private static class WorkloadGenerator {
     private final ManagedChannel channel;
-    private final List<ByteString> ids = new ArrayList<>();
+    private static final int DEFAULT_MAX_QUEUE_SIZE = 100;
+    private final Random random = new Random();
+    private final List<Task> taskValues =
+        Collections.unmodifiableList(Arrays.asList(Task.values()));
+    private final int numTasks = taskValues.size();
 
     WorkloadGenerator(ManagedChannel channel) {
       this.channel = channel;
     }
 
-    void seedData(int numAccounts) {
-      int numFailedCreateAccounts = 0;
-      for (int i = 0; i < numAccounts; i++) {
-        try {
-          ids.add(
-              WorkloadClient.getWorkloadClient(channel)
-                  .createAccount(
-                      DEFAULT_ACCOUNT_BALANCE,
-                      CreateAccountRequest.Type.CHECKING,
-                      CreateAccountRequest.Status.ACTIVE));
-        } catch (StatusRuntimeException e) {
-          numFailedCreateAccounts++;
-          logger.log(
-              Level.WARNING,
-              String.format("CreateAccount failed. Total count: %d", numFailedCreateAccounts));
+    void startSteadyLoad(int threadCount, int taskCount) {
+      ThreadPoolExecutor executor = (ThreadPoolExecutor) Executors.newFixedThreadPool(threadCount);
+      BlockingQueue<Runnable> queue = executor.getQueue();
+      while (true) {
+        if (queue.size() < DEFAULT_MAX_QUEUE_SIZE) {
+          ImmutableList<Task> tasks = generateRandomTasks(taskCount);
+          logger.log(Level.INFO, String.format("Tasks submitted %s", tasks.toString()));
```

Comment:
Nit: shouldn't you log this after submitting the task? (Also confusing that this "task" is multiple tasks.)

Reply:
1) Done
2) Are you talking about the naming of the enum in general? (name it `Tasks` instead of `Task`?)

Reply:
Didn't mean for you to make any change for (2), just noting that "task" has a different meaning for the user and reviewer.